apriltags_ros/src/apriltag_detector.cpp

```diff
@@ -131,6 +131,9 @@ void AprilTagDetector::imageCb(const sensor_msgs::ImageConstPtr& msg, const sens
     if (getTransform(output_frame_id_, cv_ptr->header.frame_id, output_transform)) {
       transform_output = true;
       header.frame_id = output_frame_id_;
+    } else {
+      ROS_WARN("Could not get transform to specied frame %s.", output_frame_id_.c_str());
```

Comment:
Should probably throttle all tf debug statements.  Can this actually happen at startup?  If not warn is fine, but should probably be relaxed on startup, then if it happens at some point unexpectedly it should be an error.  Depends on where the edge conditions exist.